Magento_Swatches/templates/product/layered/renderer.phtml

```diff
@@ -57,7 +57,7 @@ $attributeDescriptions = $helper->getAttributesDescriptions(['color']);
                 "moreInfo": {
                     "path": "/rest/V1/snowproductattributedescription/description/:attributeCode",
                     "variable": ":attributeCode",
-                    "data": "<?= $swatchCode ?>"
+                    "data": "<?= $block->escapeJs($swatchCode) ?>"
```

Comment:
test this part on frontend if it is working properly

Reply:
Where can I find it in Colibri page ? It's probably on product page but I don't see it. Maybe I should enable something in admin panel ?

Reply:
Check README in theme and section `### Product attribute description - snowdog/module-product-attribute-description`

Reply:
I don't see it but deploy changes. Can you send me link ?